Calendars/Calendars.Plugin.iOSUnified/CalendarsImplementation.cs

```diff
@@ -154,7 +154,9 @@ public async Task AddOrUpdateCalendarAsync(Calendar calendar)
                 calendar.ExternalID = deviceCalendar.CalendarIdentifier;
 
                 // Update color in case iOS assigned one
+#if !NET6_0
                 if (deviceCalendar.CGColor != null)
```

Comment:
That's interesting. Why is that `if` was needed in the first place if `CGColor` is never `null`. Maybe it's better to keep it for now, and just ignore the warning to ensure that we don't change the behavior.
Later it can re re-evaluated, if this `if` needs to be removed.

```suggestion
                if (deviceCalendar.CGColor != null!)
```

Reply:
I do suspect that `if` can be removed, and it's just that prior to net6 saying otherwise I assumed it could be null.

But your suggestion seems like the safe approach for now, which I appreciate. (also I didn't know `null!` was a thing... interesting...)